lib/msf/core/payload/windows/reverse_winhttps.rb

```diff
@@ -114,6 +97,23 @@ def required_space
     space
   end
 
+  #
+  # Get the SSL hash from the certificate, if required.
+  #
+  def get_ssl_cert_hash
```

Comment:
This looks like copy-pasta from above (in reverse_http). This should be in a new mixin to avoid duplication.